normandy/health/checks.py

```diff
@@ -21,5 +22,31 @@ def database_connected(app_configs, **kwargs):
     return errors
 
 
+def migrations_applied(app_configs, **kwargs):
+    from django.db.migrations.loader import MigrationLoader
+    errors = []
+
+    # Load migrations from disk/DB
+    loader = MigrationLoader(connection, ignore_no_migrations=True)
```

Comment:
I based this code on the [showmigrations management command](https://github.com/django/django/blob/1845bc1d1007751a7f65c66aeddc35f032f6bf41/django/core/management/commands/showmigrations.py). I didn't see the method you linked to.